cohort-engine/src/test/java/com/ibm/cohort/engine/measure/MeasureEvaluatorTest.java

```diff
@@ -33,6 +38,7 @@
 import org.junit.Test;
 import org.opencds.cqf.common.evaluation.MeasurePopulationType;
 
+import com.google.common.collect.Lists;
```

Comment:
I'm not opposed to using something else, but I didn't add this to the classpath, so don't consider it adding a dependency.  If we want to explicitly exclude guava usage probably worth a conversation and modification of the poms to prevent.

Reply:
While this is in our classpath through transitive dependencies, if we are going to use it, we need to at least declare the dependency in our POM. My preference would be to just remove it if this is the entire exposure in _our_ codebase, but I am ok with either.

Reply:
Looks like there's one other guava import in the entire project: https://github.com/Alvearie/quality-measure-and-cohort-service/blob/main/cohort-engine/src/main/java/com/ibm/cohort/engine/measure/seed/MeasureEvaluationSeeder.java#L25.  I'll add this as a topic to discuss with the wider team, it would be possible to easily remove that as well.